src/abstract.jl

```diff
@@ -46,15 +46,16 @@ mutable struct LinearOperator{T, I<:Integer, F, Ft, Fct} <: AbstractLinearOperat
   nprod::I
   ntprod::I
   nctprod::I
+  args5::Bool
```

Comment:
In `Krylov.jl`, I use `mul5` but we can find a better argument name than `args5` and `mul5`.
At least, we should use the same one.